docs/data/joy/components/stepper/ButtonStepper.tsx

```diff
@@ -22,12 +22,7 @@ export default function ButtonStepper() {
               {activeStep <= index ? index + 1 : <Check />}
             </StepIndicator>
           }
-          sx={{
-            '&::after': {
-              ...(activeStep > index &&
-                index !== 2 && { bgcolor: 'primary.solidBg' }),
-            },
-          }}
+          sx={[activeStep > index && index !== 2 && { bgcolor: 'primary.solidBg' }]}
```

Comment:
⚠️ we are missing the `&::after` selector

Reply:
This is a codemod bug, I will fix it in a separate PR.